app/controllers/ajax/inbox_controller.rb

```diff
@@ -1,25 +1,4 @@
 class Ajax::InboxController < AjaxController
-  def create
-    unless user_signed_in?
-      @response[:status] = :noauth
-      @response[:message] = t(".noauth")
-      return
-    end
-
-    question = Question.create!(content:             QuestionGenerator.generate,
-                                author_is_anonymous: true,
-                                author_identifier:   "justask",
-                                user:                current_user)
-
-    inbox = Inbox.create!(user: current_user, question_id: question.id, new: true)
-
-    @response[:status] = :okay
-    @response[:message] = t(".success")
-    @response[:success] = true
-    @response[:render] = render_to_string(partial: 'inbox/entry', locals: { i: inbox })
-    inbox.update(new: false)
```

Comment:
This is missing from the new implementation, otherwise the generated question stays read until the inbox page is refreshed again.

Maybe we can create it with `new: false` and have some "magic" variable that renders it to look like it's new

Reply:
Hmm, setting it in the partial would work, but that is kinda ugly.

Reply:
Fixed!

Reply:
works as it used to, nice!